libs/core/src/utils/sample.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { sample } from './sample';
+
+describe('@daffodil/core | sample', () => {
+  it('should return a random element of the given array', () => {
+    const initialArray = [1,2,3,4,5,6,7,8,9];
+    const result = sample(initialArray);
+    expect(initialArray).toContain(result);
+  });
```

Comment:
What happens when the array is empty? `null` or `undefined`?

Reply:
for an empty array I guess return `undefined`? I don't think it makes sense for any other value to be sampled from an empty array